marimbabot_planning/src/move_to_pose.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <ros/ros.h>
+#include <moveit/move_group_interface/move_group_interface.h>
+#include <tf2_ros/transform_listener.h>
+#include <geometry_msgs/TransformStamped.h>
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "marimba_move");
```

Comment:
Maybe chose a node name that resamples the file name in a closer way. `marimba_move` is very broad.